lib/rubocop/cli/command/auto_generate_config.rb

```diff
@@ -57,6 +60,10 @@ def line_length_cop(config)
           config.for_cop('Layout/LineLength')
         end
 
+        def options_has_only_flag?
+          @options[:only] # && !@options[:only].include?('Layout/LineLength')
```

Comment:
Why is there some commented out code here?

Reply:
just forgot to remove it - addressing this now 👍

Reply:
removed commented code